library/core/src/cell.rs

```diff
@@ -82,6 +82,21 @@
 //!
 //! The corresponding [`Sync`] version of `OnceCell<T>` is [`OnceLock<T>`].
 //!
+//! ## `LazyCell<T, F>`
+//!
+//! A common pattern with OnceCell is, for a given OnceCell, to use a specific function on every
```

Comment:
Hmm, maybe s/a specific/the same/ is easier to understand?

Reply:
Yeah, that does read better. Thanks.